pkg/errors/config.go

```diff
@@ -0,0 +1,22 @@
+package errors
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Config returns an error specifically tailored for reporting errors with configuration
+// values. You can check for these errors by calling errors.Is(err, KindConfigError)
+// (from the github.com/gomods/athens/pkg/errors package).
+//
+// Generally these kinds of errors should make Athens crash because it was configured
+// improperly
+func Config(op Op, field, helpText, url string) error {
```

Comment:
will this ever be called from outside the `config` package? why not just define it there and define it locally to minimize the API surface of this package?

Reply:
Nope. I put it here just because I think it makes more sense to group a configuration error under the `errors` package. I'm willing to be convinced otherwise 😄